doc/content/development/contributing.md

```diff
@@ -0,0 +1,127 @@
+# Contributing
+
+The contributing guidelines here are mostly adapted from
+[MOOSE contributing guidelines](https://mooseframework.inl.gov/framework/contributing.html) and
+[Commit Often, Perfect Later, Publish Once: Git Best Practices](https://sethrobertson.github.io/GitBestPractices/)
+by Seth Robertson (CC BY-SA 3.0).
+
+## Code Standards
+
+Like MOOSE, Moltres uses the ClangFormat for formatting all C++ code in the repository. If you
+ever want to contribute changes to the Moltres repository, make you run the following command
+to automatically format any new code in your current branch relative to the supplied `<branch>`
+(defaults to HEAD if omitted).
+
+```bash
+git clang-format [<branch>]
+```
+
+The configuration file for the formatting is provided in the `.clang-format` file in the base
+directory. Refer to MOOSE [Code Standards](https://mooseframework.inl.gov/sqa/framework_scs.html)
+for more details on the code standard.
+
+## Referencing Issues
+
+In every pull request, we strongly recommend referencing an issue. This practice encourages robust
+discussions on whether the change is necessary and appropriate to Moltres. It also provides
+relevant context to the new changes proposed for quicker code review.
+
+## Work in a Fork
+
+Create your own [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) where you can
+make and commit your set of changes.
+
+### 1. Fork Moltres
+
+- Navigate to [https://github.com/arfc/moltres](https://github.com/arfc/moltres)
+- Click the "Fork" button near the upper right corner of the page.
+- Clone your new fork to your local machine using the following command (replace "username" with
+  your GitHub username):
```

Comment:
The "work in a fork" section just preceeding "fork moltres" seems a little redundant. Maybe we can move the content from the "work" section into the "fork moltres" section? I agree we should be explicit about forking.